frontend/src/data/queries/query.ts

```diff
@@ -193,7 +194,7 @@ export const validateQueries = (categories: CategoryType,
                 throw new Error(`query category has unknown dimensions: ${difference}`)
             }
 	    // console.log('intersected', intersected, 'catDimensions', catDimensions)
-	    if (intersected.size == catDimensions.size) {
+	    if (intersected.size == catDimensions.size && catDimensions.size != 1) {
```

Comment:
this seems new- is this an artifact of the previous iteration?

Reply:
This is new, the query logic does not allow you to query a single category with a single dimension, it understands it as querying all dimensions for all categories and throws an error. This was not happening before because the added dimension dependency logic was adding a category to the query

Reply:
I see. Can we add a test for this new logic when we test the validate method?

Reply:
added a test to check that an error is not thrown